src/workloads/execution/ProjectSort.yml

```diff
@@ -0,0 +1,121 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Check performance of a project followed by a sort that can be swapped in order to push down. Check the 
+  performance both without and without an index.
+
```

Comment:
If this is being committed:

-  can you list the important metrics to look at.
- add some keywords  to aid finding this workload (see [here](https://github.com/mongodb/genny/blob/fd3a0f5f26bff7e13037850595a16f07f697b442/src/workloads/scale/BigUpdate.yml#L9-L15) for an example).

There is a fair bit of duplication, is it worth introducing some [GlobalDefaults](https://github.com/mongodb/genny/blob/48b3fac2e0c8c29b5f196e1263acc8d89e3762ed/src/workloads/scale/InCacheSnapshotReads.yml#L11-L14) for (assuming you are committing it):

- Collection0
- the number of repeats
- the aggregation OperationCommand

Reply:
Defaulted the collname, repeats, and the agg pipeline.